docs/using-the-compiler.rst

```diff
@@ -346,7 +346,9 @@ Input Description
       "modelCheckerSettings":
       {
         // Choose which model checker engine to use: all (default), bmc, chc, none.
-        "engine": "chc"
+        "engine": "chc",
+        // Timeout for each SMT query in milliseconds.
```

Comment:
Shouldn't there be a comment here what is the default if this is omitted?

Reply:
Ok!